packages/petr4/petr4.0.1/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "Petr4: Formal Semantics for the P4 Programming Language"
+maintainer: "jnfoster@cs.cornell.edu"
+authors: ["Nate Foster <jnfoster@cs.cornell.edu>"]
+homepage: "https://github.com/cornell-netlab/petr4"
+bug-reports: "https://github.com/cornell-netlab/petr4/issues"
+dev-repo: "git+https://github.com/cornell-netlab/petr4/"
+license: "Apache2"
+build: [
+  ["dune" "build" "--root" "." "-j" jobs "@install"]
```

Comment:
```suggestion
  ["dune" "build" "-p" name "-j" jobs]
```
This is more the usual command. Using -p allows to only select the current package and not everything that could be inside the source code

Reply:
Thanks. We've replaced this with
```
build: [
  [make]
]
```
where the `Makefile` includes a default target:
```
build:
	dune build --profile release
```
Is that acceptable?

Reply:
Thanks to some help from @smolkaj I've updated to a standard build declaration.